### PR TITLE
[WIP] Added in bullet of Python 2.4+2.5 support discontinuation

### PR DESCRIPTION
**Pull request by thaumos**
##### ISSUE TYPE
 - Docs Pull Request

##### SUMMARY
Added in bullet point about Python 2.4 support.

### CODE REVIEW

**Review by gundalow — CHANGES_REQUESTED** [review]
Needs to clarify if this is the controller or the remote end (also what are the official terms we use for the machine running ansible-playbook and the remote side?)

**Comment by gundalow** [comment]
Python 2.4 will no longer be supported for modules. This means from Ansible 2.4 (and forward) only Python 2.6+ will be supported.

**Comment by bcoca** [comment]
we skip 2.5?

File: docsite/rst/roadmap/ROADMAP_2_4.rst [hunk 1]
```diff
@@ -6,6 +6,9 @@ Ansible by Red Hat, Core 2.4
 This is meant to be a living document, and is by no means **FINAL** until
 stated otherwise in the document.
 
+- **Python 2.4 support discontinuation**
+  - Python 2.4 will no longer be supported. Going forward, Python 2.6+ will be supported.
```
Comment (bcoca) [inline docsite/rst/roadmap/ROADMAP_2_4.rst]: bikeshedding:
"Python 2.4 will no longer be supported on target machines (the controller already required >=2.6). From now on Ansible will require >=Python2.6 on targets as well."

**Comment by gundalow** [comment]
Thanks Brian, that's clearer.

For the record we *are* dropping Module support for py2.4 and py2.5

**Comment by bcoca** [comment]
then heading should also include 2.5

**Comment by thaumos** [comment]
shipit

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
[15:56] <@bcoca> i'm happy, abadger2000 is ecstatic

**Pull request merged**